Pushbullet/Configuration/PluginConfiguration.cs

```diff
@@ -19,6 +19,7 @@ public class PushbulletOptions
         public Boolean Enabled { get; set; }
         public String Token { get; set; }
         public String DeviceId { get; set; }
+        public String Channel { get; set; }
```

Comment:
It should be lowercase `string` and `bool` for these properties.
```suggestion
        public string Channel { get; set; }
```